pipeline/src/test_util.rs

```diff
@@ -18,32 +18,32 @@ pub fn resolve_test_file(file_name: &str) -> PathBuf {
     ))
 }
 
-pub fn verify_test_file<T: FieldElement>(
+pub fn verify_test_file(
     file_name: &str,
-    inputs: Vec<T>,
-    external_witness_values: Vec<(String, Vec<T>)>,
-) {
+    inputs: Vec<GoldilocksField>,
+    external_witness_values: Vec<(String, Vec<GoldilocksField>)>,
+) -> Result<(), String> {
     let pipeline = Pipeline::default()
         .from_file(resolve_test_file(file_name))
         .with_prover_inputs(inputs)
         .add_external_witness_values(external_witness_values);
     verify_pipeline(pipeline)
 }
 
-pub fn verify_asm_string<T: FieldElement>(
+pub fn verify_asm_string(
     file_name: &str,
     contents: &str,
-    inputs: Vec<T>,
-    external_witness_values: Vec<(String, Vec<T>)>,
+    inputs: Vec<GoldilocksField>,
+    external_witness_values: Vec<(String, Vec<GoldilocksField>)>,
 ) {
     let pipeline = Pipeline::default()
         .from_asm_string(contents.to_string(), Some(PathBuf::from(file_name)))
         .with_prover_inputs(inputs)
         .add_external_witness_values(external_witness_values);
-    verify_pipeline(pipeline)
+    verify_pipeline(pipeline).unwrap();
 }
 
-pub fn verify_pipeline<T: FieldElement>(pipeline: Pipeline<T>) {
+pub fn verify_pipeline(pipeline: Pipeline<GoldilocksField>) -> Result<(), String> {
```

Comment:
This function only does Pilcom verification, so I changed that it only accepts a Goldilocks pipeline.